I noticed there is a new compiler warning:
I take a closer look right now
~~~shell
In file included from /home/dash/sharif/githubDash/dart-impl/mpi/src/dart_team_group.c:14:0:
/home/dash/sharif/githubDash/dart-impl/mpi/src/dart_team_group.c: In function ‘dart_group_destroy’:
/home/dash/sharif/githubDash/dart-impl/mpi/src/dart_team_group.c:51:80: warning: pointer type mismatch in conditional expression
     DART_LOG_ERROR("Invalid group argument: %p -> %p", group, (group) ? *group : group);
                                                                                ^
/home/dash/sharif/githubDash/dart-impl/base/include/dash/dart/base/logging.h:83:40: note: in definition of macro ‘DART_LOG_ERROR’
     sn_ret = snprintf(msg_buf, maxlen, __VA_ARGS__); \

~~~

--- a/dart-impl/mpi/src/dart_team_group.c
+++ b/dart-impl/mpi/src/dart_team_group.c
@@ -48,7 +48,7 @@ dart_ret_t dart_group_destroy(
 {
 
   if (group == NULL || *group == NULL) {
-    DART_LOG_ERROR("Invalid group argument: %p -> %p", group, (group) ? *group : group);
+    DART_LOG_ERROR("Invalid group argument: %p -> %p", group, (group) ? (void*)*group : (void*)group);
     return DART_ERR_INVAL;
   }
 

